src/native/corehost/apphost/appcontainer_helpers.h

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#ifndef __APPCONTAINER_HELPERS_H__
+#define __APPCONTAINER_HELPERS_H__
+
+namespace appcontainer_helpers
+{
+    bool is_appcontainer();
+    bool is_uwp();
+    void open_url_for_appcontainer(const wchar_t* url);
```

Comment:
This should be `WCHAR`, not `wchar_t`. Even though this is only for Windows, we still aligned on `WCHAR`.